wandb/apis/public.py

```diff
@@ -4125,6 +4152,7 @@ def __init__(self, client, entity, project, name, attrs=None):
         self._is_downloaded = False
         self._dependent_artifacts = []
         self._download_roots = set()
+        self._download_logger: Optional[_ArtifactDownloadLogger] = None
```

Comment:
Why do you need an instance attribute? Couldn't find where it's used.

Reply:
Oops! Leftover from previous implementation. Thanks for the catch!